tests/test_ell1h.py

```diff
@@ -143,6 +145,20 @@ def test_J0613_STIG(toasJ0613, modelJ0613_STIG):
     f.fit_toas()
 
 
+def test_SINI_warning():
```

Comment:
Please change the name of this test to reflect the fact that it is checking for an exception.